_config.yml

```
@@ -1,4 +1,4 @@
 theme: jekyll-theme-minimal
-title: Test Testsson
-description: My portfolio website
+title: Test "Mister QA" Testsson
+description: My LinkedIn competitor
```

So yeah I don't think that this will reflect my character appropriately on the ever evolving job market. I would prefer to be referred to as a "Linkedin collaborator".
```suggestion
description: My LinkedIn collaborator
```